src/cdk-experimental/listbox/listbox.ts

```diff
@@ -297,6 +304,12 @@ export class CdkListbox implements AfterContentInit, OnDestroy, OnInit {
     } else {
       manager.onKeydown(event);
     }
+
+    /** Will select an option if shift was pressed while navigating to the option */
+    const isArrow = (keyCode === UP_ARROW || keyCode === DOWN_ARROW);
+    if (isArrow && event.shiftKey && previousActiveIndex !== this._listKeyManager.activeItemIndex) {
+      this._toggleActiveOption();
```

Comment:
Makes we wonder if this could be part of the key manager. I'm not too involved in the key manager though, so just guessing.

Reply:
From what I've seen I don't think it's part of it, but it would be a nice feature to have in the key manager for sure.

Reply:
Yeah, might be worth creating a feature issue for it (so we can keep track of it). cc. @crisbeto

Reply:
The key manager is primarily supposed to keep track of focus, it shouldn't know about what is selected.

Reply:
Oh right! For some reason I was thinking the `_toggleActiveOption` method delegates to the list key manager. Makes sense though. Resolving this discussion!